pkg/naming/labels.go

```diff
@@ -11,7 +11,7 @@ import (
 // for the given Cluster.
 func ClusterLabels(sdc *scyllav1alpha1.ScyllaDBDatacenter) map[string]string {
 	labels := ScyllaLabels()
-	labels[ClusterNameLabel] = sdc.Spec.ClusterName
+	labels[ClusterNameLabel] = sdc.Name
```

Comment:
the name of the constant is misleading now, should we change it (or have a separate one)?

Reply:
It will be misleading anyway, as value of this constant is `scylla/cluster` which we cannot change.

Reply:
Added a comment about which value should be provided.